src/Nextjs.ts

```diff
@@ -421,13 +427,31 @@ export class Nextjs extends Construct {
     const lambdaCachePolicy = cdk?.cachePolicies?.lambdaCachePolicy ?? this.createCloudFrontLambdaCachePolicy();
 
     // requests for static objects
+    const defaultStaticMaxAge = cdk?.cachePolicies?.staticClientMaxAgeDefault;
+    const staticResponseHeadersPolicy = defaultStaticMaxAge
```

Comment:
Is `0` a valid value? If so you'll want to change the conditional check.

Reply:
fixed